krkn/scenario_plugins/syn_flood/syn_flood_scenario_plugin.py

```diff
@@ -137,3 +150,21 @@ def is_node_affinity_correct(self, obj) -> bool:
 
     def get_scenario_types(self) -> list[str]:
         return ["syn_flood_scenarios"]
+
+    @staticmethod
+    def rollback_syn_flood_pods(rollback_content: RollbackContent, lib_telemetry: KrknTelemetryOpenshift):
+        """
+        Rollback function to delete syn flood pods.
+
+        :param rollback_content: Rollback content containing namespace and resource_identifier.
+        :param lib_telemetry: Instance of KrknTelemetryOpenshift for Kubernetes operations
+        """
+        try:
+            namespace = rollback_content.namespace
+            pod_names = list(rollback_content.resource_identifier)
```

Comment:
`2025-11-18 10:59:17,009 [INFO] Rolling back syn flood pods: ['[', "'", 's', 'y', 'n', '-', 'f', 'l', 'o', 'o', 'd', '-', 'p', 'a', 'e', 'w', 'q', 'r', 'o', 'n', 'p', 'g', "'", ',', ' ', "'", 's', 'y', 'n', '-', 'f', 'l', 'o', 'o', 'd', '-', 'e', 'm', 'x', 'x', 'y', 'v', 'g', 'z', 'u', 't', "'", ']'] in namespace: default
2025-11-18 10:59:17,269 [INFO] Rollback of syn flood pods completed successfully`

You're not deserializing in the correct way